actions.js

```diff
@@ -372,8 +372,8 @@ module.exports = class Action {
     
     getLatestBlockHeight(){          
         return this._blockchainHandler().getLatestBlockHeight()
-        .then(latestBlockHeight => {
-            logger.console("Ltest block height :" + latestBlockHeight);
+        .then(data => {
+            logger.console("Latest block height :" + data.body.result.Block.header.height);
```

Comment:
We don't need to know about the structure of the data here, so it's better we just get the latest block height from the getLatestBlockHeight()